Nodejs/Product/Nodejs/TypingsAquisitionTool/index.js

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Seems that npm requires some sort of index/main file, so I just created this empty one.